---
output:
  md_document:
    variant: markdown_github
bibliography: bibliography_V2.bib
---

# [NFDI4Microbiota - Metadata Standards]{#N4Mintroduction} <br><br>

<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />
This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.

The primary objective of this GitHub page is to serve as **a centralized repository for existing (meta)data standards**. The purpose is to provide the international microbiological community with a **comprehensive and easily accessible compilation of established standards**, facilitating efficient navigation and utilization for researchers involved in collecting and submitting (meta)data to public repositories.

In line with the [**NFDI4Microbiota**](https://nfdi4microbiota.de/){#nfdi4microbiotastandardsandpolicies} [@nfdi] [@nfdi_microbiota] project's objectives, this page aims to address the challenges of microbial (meta)data accessibility and consistency. The efficient exchange of usable information between research groups, sequencing centers, and data repositories has been a long-standing issue. Measure 2.1 (M2.1) specifically focuses on maximizing data quality within the [NFDI4Microbiota](https://nfdi4microbiota.de/) [@nfdi_microbiota] system by enforcing compliance with existing standards and identifying additional tailored data standards and metadata requirements.

[**Goals:**]{#goals} By centralizing standard parameters for metadata, the project ensures that generated data is reproducible and comparable both spatially and temporally. To achieve this, two **milestones** have been set:

- defining data standards for different types of **raw data**, and ensuring their quality and reliability 
- defining data standards for **technical metadata**, further enhancing the consistency and usability of the collected metadata. <br><br>

In the context of metadata quality standards in microbial science, two main categories are being considered: 

- [**Technical**](#technical-metadata-standards)
- [**Biological/Environmental**](#bio-env-metadata-standards) <br><br>

These categories aim to encompass the necessary information that researchers collecting and submitting metadata to public repositories need to provide. By adhering to these standardized metadata categories, researchers can ensure the integrity and interoperability of their data, enabling effective collaboration and comparative analysis within the international microbiological community.


## Reading this Github

- Begin by reading the **[NFDI4Microbiota introduction](#N4Mintroduction), [Standards and Policies](#nfdi4microbiotastandardsandpolicies)** information, and **[Goals](#goals)**
- Next, read the information regarding [**technical metadata standards section**](#technical-metadata-standards)
- Third, read the [**biological/environmental metadata standards section**](#biological-and-environmental-metadata-standards) <br><br>

[**Figure 1.**](#figure1) Outlines the key aspects considered for determining minimal metadata standards that can be universally applicable across various datasets and microbiomes. These aspects encompass both technical and biological/environmental (Bio/Env) considerations. The figure illustrates the comprehensive approach used to establish minimal metadata standards for diverse research settings by combining already established standards for differing data types and biomes. <br><br> 

<!---
For making the access link for other people not on the collaborator list, it should be better to use the relative address instead of the constant one.
Thus, I replace the "https://github.com/mdsufz/NFDI4Microbiota_MetadataStandards/blob/main" with "."
-->

![Overview](./images/Overview24June2022.jpg){#figure1}
**Figure 1. Flow Chart of Technical and Biological/Environmental Metadata Standard Development**

This flow chart illustrates the process of developing metadata standards for both Technical and Biological/Environmental aspects. Technical parameters are categorized based on data types, while Bio/Env parameters are organized according to biome types. Additionally, specific considerations, such as file type and host, are taken into account to enhance the comprehensiveness of the standards. <br><br>

# [Technical Metadata Standards]{#technical-metadata-standards} <br><br>

## Technical metadata section 
### 1. Data types

The following data types were considered when establishing minimal **technical** metadata standards for M2.1:

- Genomes
- Amplicon
- Metagenomes
- Metagenome assembled genomes (MAGs)
- Transcriptomes
- Metatranscriptomes
- Proteomes
- Metaproteomes
- Metabolomes<br><br>

Standard parameter considerations for FASTQ and FASTA formats are displayed in [**Figure 2.**](#figure2) and [**Figure 3.**](#figure3), respectively. Parameter applicability to different data types and the time of data generation (i.e., before sequencing or during data processing) are shown on the left and right, respectively.


Additionally, standards are being considered for [**data transfer and data integrity**](#data-transfer-data-integrity) to ensure quality is maintained throughout various processes of data file exchange. <br><br>

### 2. Overview of minimal technical FASTQ and FASTA metadata considerations. <br><br>

![FASTQMetadataTablesOverview](./images/MinimalTechnicalTable_FASTQ.jpg){#figure2}
**Figure 2. Overview of Minimal Technical Metadata for FASTQ Files ** 

This figure provides an overview of the minimal technical metadata relevant to FASTQ files. The left side lists the applicability of parameters to different data types, such as (meta)genome, (meta)transcriptome, etc. On the right side, the time of metadata generation is indicated. <br><br>


![FASTAMetadataTablesOverview](./images/MinimalTechnicalTable_FASTA.jpg){#figure3} 
**Figure 3. Overview of Minimal Technical Metadata for FASTA Files ** 

This figure presents an overview of the minimal technical metadata relevant to FASTA files. On the left side, the applicability of parameters to different data types, including (meta)genome, (meta)transcriptome, etc., is listed. The right side provides information about the time of metadata generation. <br><br>



### 3. Minimal technical metadata by technology and file type 

Establishing a file-specific metadata standard list poses a significant challenge due to variations in file types across instruments used in metabolomic and proteomic analyses. Thus, researchers can find the metadata standards for each specific technology within corresponding links. This approach recognizes the complexities of defining comprehensive and universally applicable metadata standards that differ based on technology.  

- 2.1 [Genome Sequencing](./Technical/Genome_Technical_Metadata.md)
  - Genomic FASTQ
  - Genomic FASTA
- 2.2 [Amplicon Sequencing](./Technical/Amplicon_Technical_Metadata.md)
  - Amplicon FASTQ
- 2.3 [Metagenome Sequencing](./Technical/Metagenome_Technical_Metadata.md)
  - Metagenome FASTQ
  - Metagenome FASTA
  - Metagenome assembled genome (MAG) FASTA
- 2.4 [Transcriptome Sequencing](./Technical/Transcriptome_Technical_Metadata.md)
  - Transcriptome FASTQ
  - Transcriptome FASTA
- 2.5 [Metatranscriptome Sequencing](./Technical/Metatranscriptome_Technical_Metadata.md)
  - Metatranscriptome FASTQ
  - Metatranscriptome FASTA
- 2.6 [Proteome sequencing](./Technical/Proteome_Technical_Metadata.md)
  - Proteome
  - Proteome - experimental protocol edition
- 2.7 Metaproteome sequencing 
- 2.8 [Metabolome sequencing](./Technical/Metabolome_Technical_Metadata.md)
  - Metabolome
  - Metabolome - experimental protocol edition
- 2.9 [BIOM or tabular files](./Technical/BIOM_or_Tabular_Technical_Metadata.md) <br><br>



### 4. Data transfer and data integrity{#data-transfer-data-integrity}

The work of the [Data transfer and data integrity](./Technical/Data_Transfer_Data_Integrity.md) section focuses on:

- Examples of existing data transfer & data integrity checks
- Data integrity considerations by file type <br><br>


# [Biological and Environmental Metadata Standards]{#bio-env-metadata-standards} <br><br>

## Bio/Env metadata section 
### 1. Biomes considered

Six microbiomes were considered to compile a minimal set of biological and environmental metadata standards. Environmental and biological parameters were identified as minimums applicable to individual biomes and/or hosts.

The Minimal **Biological and Environmental** microbiome metadata standards within M2.1 were collected to apply to the following biomes:

- [Marine](./Biological_Environmental/Marine_BioEnv_Metadata.md)
- [Terrestrial](./Biological_Environmental/Terrestrial_BioEnv_Metadata.md)
- [Terrestrial (constructed)](./Biological_Environmental/TerrestrialConstructed_BioEnv_Metadata.md)
- [Plant-associated](./Biological_Environmental/PlantAssoc_BioEnv_Metadata.md)
- [Animal-associated](./Biological_Environmental/AnimalAssoc_BioEnv_Metadata.md)
- [Human-associated](./Biological_Environmental/Human_BioEnv_Metadata.md)
- Microbe-associated

Tentative standard minimal biological and environmental parameter considerations are displayed in [**Figure 4**](#figure4). Parameter applicability to different biomes are shown on the left axis. <br><br>

![BioEnvMetadata23June2022](./images/BioEnvMetadata9Aug2023.jpg){#figure4}
**Figure 4. Tentative Minimal Biological and Environmental Metadata**. 

This figure presents the division of minimal biological and environmental metadata into distinct categories. Site metadata includes specifications and environmental parameters related to the geographic sampling location, while sample material and host metadata provide information specific to host-associated systems. The applicability of these standards to different microbiomes is shown on the left. Additionally, conditional metadata standards encompass pertinent minimal cultivation information.

The references in the figure are from the following sources: 

- **Marine references**: 
  - GSC MIxS: Water MIMS [@gsc_water_mims]
  - ENA MMC: ENA Checklist: Marine Microalgae [@ena_marine_microalgae]
  - ENA Tara Oceans; Checklist: ERC000030 [@ena_tara_oceans]
  - GSC Minimum Information about any (x) Sequence (MIxS); ENA checklist: Water environment [@gsc_mixs_water]
  - The environment ontology: contextualising biological and biomedical entities [@envo_ontology]
  - The minimum information about a genome sequence (MIGS) specification [@field_migs]
  - Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications [@yilmaz2011]
  - A standard MIGS/MIMS compliant XML Schema: Toward the development of the Genomic Contextual Data Markup Language (GCDML) [@kottmann_gcdml]
  - Standard reporting requirements for biological samples in metabolomics experiments: environmental context [@morrison2007]
- **Terrestrial / Terrestrial(constructed)**
  - GSC MIxS: Miscellaneous Natural Or Artificial Environment MIMS [@gsc_misc_env_mims]
  - GSC MIxS: Sediment MIMS [@gsc_sediment_mims]
  - GSC MIXS: Soil MIMS [@gsc_soil_mims]
  - GSC MIxS: Wastewater Sludge MIMS [@gsc_wastewater_sludge_mims]
  - GSC MIxS: Built Environment MIMS [@gsc_built_env_mims]
  - The environment ontology: contextualising biological and biomedical entities [@envo_ontology]
  - The minimum information about a genome sequence (MIGS) specification [@field_migs]
  - Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications [@yilmaz2011]
  - A standard MIGS/MIMS compliant XML Schema: Toward the development of the Genomic Contextual Data Markup Language (GCDML) [@kottmann_gcdml]
  - Standard reporting requirements for biological samples in metabolomics experiments: environmental context [@morrison2007]
- **Plant-associated**
  - GSC MIxS: Plant-associated MIMS [@gsc_plant_assoc_mims]
  - GSC MIxS: Agriculture MIMS [@gsc_agriculture_mims]
  - GSC MIxS: Symbiont-associated MIMS [@gsc_symbiont_assoc_mims]
  - ENA MMC: ENA  Checklist: Marine Microalgae [@ena_marine_microalgae]
  - The environment ontology: contextualising biological and biomedical entities [@envo_ontology]
  - The minimum information about a genome sequence (MIGS) specification [@field_migs]
  - Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications [@yilmaz2011]
  - A standard MIGS/MIMS compliant XML Schema: Toward the development of the Genomic Contextual Data Markup Language (GCDML) [@kottmann_gcdml]
  - Standard reporting requirements for biological samples in metabolomics experiments: environmental context [@morrison2007]
- **Animal-associated**
  - GSC MIxS: Host-associated MIMS [@gsc_host_assoc_mims]
  - The environment ontology: contextualising biological and biomedical entities [@envo_ontology]
  - The minimum information about a genome sequence (MIGS) specification [@field_migs]
  - Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications [@yilmaz2011]
  - A standard MIGS/MIMS compliant XML Schema: Toward the development of the Genomic Contextual Data Markup Language (GCDML) [@kottmann_gcdml]
  - Standard reporting requirements for biological samples in metabolomics experiments: environmental context [@morrison2007]
- **Human-associated**
  - MIMS: metagenome/environmental, human-associated; version 6.0 Package [@mims_package] 
  - GSC MIxS human associated; ENA Checklist: ERC000014 [@gsc_mixs_human_associated]
  - GSC MIxS: Human-associated MIMS [@gsc_human_assoc_mims]
  - GSC MIxS: Human-gut MIMS [@gsc_human_gut_mims]
  - GSC MIxS: Human-oral MIMS [@gsc_human_oral_mims]
  - GSC MIxS: Human-skin MIMS [@gsc_human_skin_mims]
  - GSC MIxS: Human-vaginal MIMS [@gsc_human_vaginal_mims]
  - The environment ontology: contextualising biological and biomedical entities [@envo_ontology]
  - The minimum information about a genome sequence (MIGS) specification [@field_migs]
  - Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications [@yilmaz2011]
  - A standard MIGS/MIMS compliant XML Schema: Toward the development of the Genomic Contextual Data Markup Language (GCDML) [@kottmann_gcdml]
  - Standard reporting requirements for biological samples in metabolomics experiments: environmental context [@morrison2007]
  - U.S. Office of Management and Budget (OMB): About the Topic of Race [@us_omb_race]
- **Microbe-associated**
  - GSC MIxS: Miscellaneous Natural Or Artificial Environment MIMS [@gsc_misc_env_mims]
  - GSC MIxS: Sediment MIMS [@gsc_sediment_mims]
  - GSC MIXS: Soil MIMS [@gsc_soil_mims]
  - GSC MIxS: Wastewater Sludge MIMS [@gsc_wastewater_sludge_mims]
  - GSC MIxS: Microbial Mat Biofilm MIMS [@gsc_microbial_mat_biofilm_mims]
  - The environment ontology: contextualising biological and biomedical entities [@envo_ontology]
  - The minimum information about a genome sequence (MIGS) specification [@field_migs]
  - Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications [@yilmaz2011]
  - A standard MIGS/MIMS compliant XML Schema: Toward the development of the Genomic Contextual Data Markup Language (GCDML) [@kottmann_gcdml]
  - Standard reporting requirements for biological samples in metabolomics experiments: environmental context [@morrison2007]
  - Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea [@bowers2017]
  - Roadmap for naming uncultivated Archaea and Bacteria [@murray2020]
<br><br>

### 2. Data/metadata categorization

The categorization framework in [**Figure 5**](#figure5) should be considered when determining the applicable metadata standards for each dataset. This framework can serve as a valuable tool for connecting information about samples from marine, terrestrial, or engineered systems. Additionally, it facilitates the inclusion of cultivated samples, whether they were cultured from a commercially-available source or isolated from an environmental sample by the user.


To enhance searchability in downstream analyses, users can select multiple environment categories if relevant. For instance, they may choose both "marine" and "terrestrial" for a tidal flat site, "engineered" and "terrestrial" for a greenhouse agricultural site, or "engineered" and "marine" for a commercially-available culture initially isolated from the ocean. <br><br>

#### **Category Flow Chart**
![](./images/CategoryFlowchart.JPG){#figure5}

**Figure 5. Tentative Categorization Framework for Biological/Environmental Metadata Requirements **

This figure showcases a preliminary categorization framework to establish minimal biological/environmental metadata requirements. The framework connects host-associated systems to marine, terrestrial, or engineered environments while enabling effective tracking of data affiliated with cultivated samples. The structure should provide valuable insights for organizing and comprehensively accessing diverse datasets. <br><br>

**Figures 6 - 8** show examples of minimal biological/environmental metadata applicability to different sample categorizations. <br><br>


#### **Human Gut Example**
![](./images/CategoryFrameworkSlides_HumanGutExample_mb.gif)
**Figure 6. Example of Categorizing a Human Gut-Associated and Cultivated Sample with Applicable Minimal Metadata ** 

This figure provides an illustrative example of the categorization process for a human gut-associated and cultivated sample. It showcases the minimal metadata that are applicable and relevant for this specific sample type. <br><br>

#### **Tidal Flat Example**
![](./images/CategoryFrameworkSlides_TidalFlatExample_mb.gif)
**Figure 7. Example of Categorizing a Tidal Flat and Cultivated Sample with Applicable Minimal Metadata ** 

This figure presents a practical example of categorizing a tidal flat cultivated sample, along with the relevant minimal metadata. The illustration demonstrates how the proposed framework accommodates overlapping environments, such as terrestrial and marine, specifically for intertidal regions.<br><br>


#### **Lab Culture Example**
![](./images/CategoryFrameworkSlides_LabCultureExample_mb.gif)
**Figure 8. Example of Categorizing a Known Lab Cultured Sample with Applicable Minimal Metadata ** 

This figure presents an example of categorizing a known lab-cultured sample, along with the corresponding minimal metadata. The bidirectionality of the categorization framework is highlighted, as it enables the linkage between known, commercially available cultures and their original sample environments. <br><br>

# References