---
output:
  md_document:
    variant: markdown_github
bibliography: ../bibliography_V2.bib
---

## 2.2 Amplicon sequencing

A searchable and exportable tab-separated [table](Amplicon_Technical_Metadata.tsv) of the following metadata is now available.

## Minimal technical metadata for `Amplicon FASTQ` data
|**metadata**| **definition/examples** | **source** |
|------------|-------------------------|------------|
| sample_name | Identifier of the sample |**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria] |
| seq_meth | Sequencing method used (e.g. Illumina HiSeq 2000)|**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria], **ENA Metadata Validation: Instrument** [@ena_metadata_instrument]|
| lib_layout | Single-end, paired end or others |**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria]|
| lib_source | Genomic, metagenomic, transcriptomic, etc. |**ENA Metadata Validation: Source** [@ena_metadata_source]|
| lib_strategy | WGS, Amplicon, etc. |**ENA Metadata Validation: Strategy** [@ena_metadata_strategy]|
| lib_selection | Random, PCR, etc. |**ENA Metadata Validation: Selection** [@ena_metadata_selection]|
| sequence_count | Number of reads in the library (sequencing depth) or 'spots' |**Adapted from NCBI-SRA** [@leinonen2011sequence]|
| basepairs_count | Number of base pairs (nucleotides) in the library or 'bases' |**Adapted from NCBI-SRA** [@leinonen2011sequence]|
| average_length | As basepairs_count divided by sequence_count |**Calculated as basepairs_count/sequence_count**|
| sequence_count_qual | Number of reads in the library (sequencing depth) after quality filtering |**SRA-Tinder** [@sra_tinder]|
| basepairs_count_qual | Number of base pairs (nucleotides) in the library after quality filtering |**SRA-Tinder** [@sra_tinder]|
| target_gene | Targeted gene name for marker gene studies (e.g. 16S rRNA, ITS) |**GSC MIXS: MIMARKSSpecimen** [@gsc_mixs_mimarks_specimen]|
| target_subfragment | Name of the targeted subregion (e.g. V3-V4, ITS1) |**GSC MIXS: MIMARKSSpecimen** [@gsc_mixs_mimarks_specimen]|
| pcr_primers | Forward and reverse primer sequences used |**GSC MIXS: MIMARKSSpecimen** [@gsc_mixs_mimarks_specimen]|
| checksum | Hash value (e.g. MD5) for data integrity |[Data transfer and data integrity](Data_Transfer_Data_Integrity.md)|

|Comments/questions: |
|--------------------|
|Should we also add nucl_ext_method and nucl_extr_treat here -MB 11AUG23|

The publications describing the reasons for formation of **The minimum information about a genome sequence (MIGS)** [@field_migs] and **Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications** [@yilmaz2011] can be found online.

# References

