---
output:
  md_document:
    variant: markdown_github
bibliography: ../bibliography_V2.bib
---

## 2.1 Genome sequencing

A searchable and exportable tab-separated [table](Genome_Technical_Metadata.tsv) of the following metadata is now available.

## Minimal technical metadata for `Genomic FASTQ` data

|**metadata**| **definition/examples** | **source** |
|------------|-------------------------|------------|
| sample_name | Identifier of the sample |**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria]|
| seq_meth | Sequencing method used (e.g. Illumina HiSeq 2000)|**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria], **ENA Metadata Validation: Instrument** [@ena_metadata_instrument]|
| lib_layout | Single-end, paired end or others |**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria]|
| lib_source | Genomic, metagenomic, transcriptomic, etc. |**ENA Metadata Validation: Source** [@ena_metadata_source]|
| lib_strategy | WGS, Amplicon, etc. |**ENA Metadata Validation: Strategy** [@ena_metadata_strategy]|
| lib_selection | Random, PCR, etc. |**ENA Metadata Validation: Selection** [@ena_metadata_selection]|
| sequence_count | Number of reads in the library (sequencing depth) or 'spots' |**Adapted from NCBI-SRA** [@leinonen2011sequence]|
| basepairs_count | Number of base pairs (nucleotides) in the library or 'bases' |**Adapted from NCBI-SRA** [@leinonen2011sequence]|
| average_length | As basepairs_count divided by sequence_count |**Calculated as basepairs_count/sequence_count**|
| sequence_count_qual | Number of reads in the library (sequencing depth) after quality filtering |**SRA-Tinder** [@sra_tinder]|
| basepairs_count_qual | Number of base pairs (nucleotides) in the library after quality filtering |**SRA-Tinder** [@sra_tinder]|
| checksum | Hash value (e.g. MD5) for data integrity |[Data transfer and data integrity](Data_Transfer_Data_Integrity.md)|

|Comments/questions: |
|--------------------|
|Should we also add nucl_ext_method and nucl_extr_treat here -MB 11AUG23|

## Minimal technical metadata for `Genomic FASTA` data

|**metadata**| **definition/examples** | **source** |
|------------|-------------------------|------------|
| run_ref | Accessions/identifiers linking to the raw data (FASTQ) |**Adapted from ENA** [@ena_submit_read_alignments]|
| tax_ident | Phylogenetic marker(s) for MAG assignation |**GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| assembly_qual | Assembly quality category (e.g. Medium Quality Draft) |**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria]|
| assembly_software | Tool(s) used, version and parameters |**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria]|
| coverage | The estimated depth of sequencing coverage (in x) |**ENA Submitting Metagenome Assemblies** [@ena_metagenome_assemblies]|
| number_contig | Total number of contigs |**GSC MIxS/MIGS Bacteria** [@gsc_migs_bacteria], **Roadmap for naming uncultivated Archaea and Bacteria** [@murray2020]|
| N50 | The length of the shortest contig representing half of the assembly length| **Roadmap for naming uncultivated Archaea and Bacteria** [@murray2020]|
| LSU_recover | Detection of the 23S rRNA (BA) or 5.8S/28S rRNA (E) |**Adapted from GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| LSU_recover_software | Tools for LSU extraction |**Adapted from GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| SSU_recover | Detection of the 16S rRNA (BA) or 18S rRNA (E) |**Adapted from GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| SSU_recover_software | Tools for SSU extraction |**Adapted from GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| trnas | Total number of tRNAs identified |**GSC MIXS: MIMAG** [@gsc_mixs_mimag], **Roadmap for naming uncultivated Archaea and Bacteria** [@murray2020]|
| trna_ext_software | Tools used for tRNA identification |**GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| compl_score | Completeness score |**GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| compl_software | Tools used for completion estimate |**GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| contam_score | Contamination score |**GSC MIXS: MIMAG** [@gsc_mixs_mimag]|
| contam_software| Tool(s) used in contamination screening |**GSC MIXS: MIMAG** [@gsc_mixs_mimag]|

The publications describing the reasons for formation of **The minimum information about a genome sequence (MIGS)** [@field_migs], **Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications** [@yilmaz2011], **Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea** [@bowers2017], and **Roadmap for naming uncultivated Archaea and Bacteria** [@murray2020] can be found online.

We also encourage our readers to have a look at the **HumanMetagenomeDB: a public repository of curated and standardized metadata for human metagenomes** [@kasmanas2021] and **TerrestrialMetagenomeDB: a public repository of curated and standardized metadata for terrestrial metagenomes** [@correa2020] to better understand some of the metadata fields that can be found in the tables above.

# References