## 2.6 Proteomics 
## &emsp; Minimal technical metadata for proteomics
|**metadata**| **definition/examples** | **source** |
|------------|-------------------------|------------|
| Project title | Name of the project | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Project description | Overall description of the project | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Contact information | person(s) responsible for contact including their affiliation | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28], MIAPE[^29] |
| Sample species | Species of the sample | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| PX_ID | Unique dataset identifier | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Instrument | Mass spectrometer & manufacturer | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], mage-tab[^26], iprox[^27], jpost[^28] |
| Modifications | What post-translational modifications were done if applicable | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], mage-tab[^26], iprox[^27], jpost[^28] |
| File type | Type of file uploaded (e.g. RAW, SEARCH, RESULT, etc.) | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Ion source | ESI, MALDI or other ionisation source including relevant parameters | MIAPE[^29] |
| DOI | ID number of the publication associated with the dataset | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27] |
| Post source component | Analyzer & activation/dissociation type | MIAPE[^29] |

|Comments/questions: |
|--------------------|
| |

## &emsp; Minimal technical metadata for proteomics - experimental protocol edition
|**metadata**| **definition/examples** | **source** |
|------------|-------------------------|------------|
| &#x1F539; _Sample processing protocol_ | short description of sample preparation including extraction, separation, enrichment, clean-up, digestion and mass spectrometry protocols | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28], MIAPE[^29] |
| &#x1F539; _Data processing protocol_ | short description of bioinformatics pipeline, software tools including versions, main search parameters and quantitative analysis | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28], MIAPE[^29] |
| Growth protocol | description of the growth protocol or organism preparation | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb, mage-tab[^26], iprox[^27], jpost[^28] |
| Treatment protocol | description of the treatments applied to the organism prior to sample acquisition | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Extraction protocol | description of the extraction of proteins from the treated sample(s) | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Separation protocol | description of the separation of proteins and/or peptides | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Digestion protocol | description of the digestion of the proteins into peptides | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Acquisition protocol | description of the acquisition of mass spectra from the peptide sample | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |
| Protocol parameters | list of parameter names | Pride[^22], proteomeX[^23], peptideatlas[^24], massive[^25], gigadb[^30], mage-tab[^26], iprox[^27], jpost[^28] |

|Comments/questions: |
|--------------------|
|&#x1F539; Discussion question: one or two fields for protocols or more? -SS 02May2022 |


# References
[^1]:Field, D., Garrity, G., Gray, T., Morrison, N., Selengut, J., Sterk, P., Tatusova, T., Thomson, N., Allen, M. J., Angiuoli, S. V., Ashburner, M., Axelrod, N., Baldauf, S., Ballard, S., Boore, J., Cochrane, G., Cole, J., Dawyndt, P., De Vos, P., DePamphilis, C., … Wipat, A. (2008). The minimum information about a genome sequence (MIGS) specification. Nature biotechnology, 26(5), 541–547. https://doi.org/10.1038/nbt1360, https://gensc.org/mixs/,  
https://gensc.org/publications-2/

[^2]:Kasmanas, J. C., Bartholomäus, A., Corrêa, F. B., Tal, T., Jehmlich, N., Herberth, G., von Bergen, M., Stadler, P. F., de Carvalho, A. & da Rocha, U. N. (2021). HumanMetagenomeDB: a public repository of curated and standardized metadata for human metagenomes. Nucleic Acids Research, 49(D1), D743–D750., https://webapp.ufz.de/hmgdb/

[^3]:Corrêa, F. B., Saraiva, J. P., Stadler, P. F. & da Rocha, U. N. (2020). TerrestrialMetagenomeDB: a public repository of curated and standardized metadata for terrestrial metagenomes. Nucleic Acids Research, 48(D1), D626-D632., https://webapp.ufz.de/tmdb/

[^4]:Bowers, R., Kyrpides, N., Stepanauskas, R. et al. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. Nat Biotechnol 35, 725–731 (2017). https://doi.org/10.1038/nbt.3893, https://gensc.org/mixs/

[^5]: Murray, A.E., Freudenstein, J., Gribaldo, S. et al. Roadmap for naming uncultivated Archaea and Bacteria. Nat Microbiol 5, 987–994 (2020). https://doi.org/10.1038/s41564-020-0733-x, https://www.isme-microbes.org/seqcode-initiative

[^6]: Yilmaz, Pelin et al. “Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications.” Nature biotechnology vol. 29,5 (2011): 415-20. doi:10.1038/nbt.1823,  https://gensc.org/mixs/

[^7]: Shakya, Migun et al. “Advances and Challenges in Metatranscriptomic Analysis.” Frontiers in genetics vol. 10 904. 25 Sep. 2019, doi:10.3389/fgene.2019.00904

[^8]: https://ena-docs.readthedocs.io/en/latest/faq/runs.html#

[^9]: https://www.ncbi.nlm.nih.gov/geo/info/seq.html

[^10]: https://anonsvn.ncbi.nlm.nih.gov/repos/v1/trunk/sra/doc/SRA_1-1/SRA_Quick_Start_Guide.pdf

[^11]: https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc&f=vdb-validate

[^12]: Alex L Mitchell, Alexandre Almeida, Martin Beracochea, Miguel Boland, Josephine Burgin, Guy Cochrane, Michael R Crusoe, Varsha Kale, Simon C Potter, Lorna J Richardson, Ekaterina Sakharova, Maxim Scheremetjew, Anton Korobeynikov, Alex Shlemov, Olga Kunyavskaya, Alla Lapidus, Robert D Finn, MGnify: the microbiome analysis resource in 2020, Nucleic Acids Research, Volume 48, Issue D1, 08 January 2020, Pages D570–D578, https://doi.org/10.1093/nar/gkz1035

[^13]: The Metagenomics RAST server — A public resource for the automatic phylogenetic and functional analysis of metagenomes
F. Meyer, D. Paarmann, M. D’Souza, R. Olson , E. M. Glass, M. Kubal, T. Paczian, A. Rodriguez, R. Stevens, A. Wilke, J. Wilkening, and R. A. Edwards
BMC Bioinformatics 2008, 9:386,  https://help.mg-rast.org/user_manual.html#data-hygiene

[^14]: Bassi, S., Gonzalez, V. New checksum functions for Biopython. Nat Prec (2007). https://doi.org/10.1038/npre.2007.278.1

[^15]: Babnigg, G. and Giometti, C.S. (2006), A database of unique protein sequence identifiers for proteome studies. Proteomics, 6: 4514-4522. https://doi.org/10.1002/pmic.200600032

[^16]: https://www.ncbi.nlm.nih.gov/genbank/tsa/

[^17]:  Athar A. et al., 2019. ArrayExpress update - from bulk to single-cell expression data. Nucleic Acids Res, doi: 10.1093/nar/gky964. Pubmed ID 30357387.  https://www.ebi.ac.uk/arrayexpress/

[^18]:The FAIR Cookbook: a deliverable of the FAIRplus project (grant agreement 802750), funded by the IMI programme, a private-public partnership that receives support from the European Union’s Horizon 2020 research and innovation programme and EFPIA Companies.   https://faircookbook.elixir-europe.org/content/recipes/interoperability/transcriptomics-metadata.html

[^19]:https://www.nfdi.de/

[^20]:https://nfdi4microbiota.de/

[^21]:Hedlund et al. (In review) https://disc-genomics.uibk.ac.at/seqcode//files/Hedlund_et_al.pdf

[^22]:(https://www.ebi.ac.uk/pride/markdownpage/submitdatapage)

[^23]:(http://www.proteomexchange.org/docs/guidelines_px.pdf)

[^24]:(http://www.peptideatlas.org/)

[^25]:(https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp)

[^26]:(https://psidev.info/magetab)

[^27]:(https://www.iprox.cn/page/helpEn.html#pag5)

[^28]:(https://repository.jpostdb.org/help)

[^29]:(https://www.psidev.info/sites/default/files/2018-03/MIAPE_MS_2.98.pdf)

[^30]:(http://gigadb.org/site/guide)

[^31]:(https://www.ebi.ac.uk/metabolights/index)

[^32]:(https://github.com/MSI-Metabolomics-Standards-Initiative/CIMR)

[^33]:(http://metabolonote.kazusa.or.jp/Main_Page)

[^34]:(https://www.metabolomicsworkbench.org/)

[^35]:(http://services.cbib.u-bordeaux.fr/MERYB/home/home.php)
