---
output:
  md_document:
    variant: markdown_github
bibliography: ../bibliography_V2.bib
---

## 2.6 Proteomics 

A searchable and exportable tab-separated [table](Proteome_Technical_Metadata.tsv) of the following metadata is now available.

## Minimal technical metadata for proteomics
|**metadata**| **definition/examples** | **source** |
|------------|-------------------------|------------|
| Project title | Name of the project |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Project description | Overall description of the project |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Contact information | person(s) responsible for contact including their affiliation |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines], **MIAPE** [@taylor2007miape]|
| Sample species | Species of the sample |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| PX_ID | Unique dataset identifier |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Instrument | Mass spectrometer & manufacturer |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Modifications | What post-translational modifications were done if applicable |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| File type | Type of file uploaded (e.g. RAW, SEARCH, RESULT, etc.) |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Ion source | ESI, MALDI or other ionisation source including relevant parameters |**MIAPE** [@taylor2007miape]|
| DOI | ID number of the publication associated with the dataset |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Post source component | Analyzer & activation/dissociation type |**MIAPE** [@taylor2007miape]|

|Comments/questions: |
|--------------------|
| |

## &emsp; Minimal technical metadata for proteomics - experimental protocol edition
|**metadata**| **definition/examples** | **source** |
|------------|-------------------------|------------|
| Sample processing protocol | short description of sample preparation including extraction, separation, enrichment, clean-up, digestion and mass spectrometry protocols |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines], **MIAPE** [@taylor2007miape]|
| Data processing protocol | short description of bioinformatics pipeline, software tools including versions, main search parameters and quantitative analysis |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines], **Adapted from MIAPE** [@taylor2007miape]|
| Growth protocol | description of the growth protocol or organism preparation |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Treatment protocol | description of the treatments applied to the organism prior to sample acquisition |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Extraction protocol | description of the extraction of proteins from the treated sample(s) |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Separation protocol | description of the separation of proteins and/or peptides |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Digestion protocol | description of the digestion of the proteins into peptides |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Acquisition protocol | description of the acquisition of mass spectra from the peptide sample |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|
| Protocol parameters | list of parameter names |**ProteomeXchange Consortium Guidelines** [@proteomexchange_guidelines]|

|Comments/questions: |
|--------------------|
|&#x1F539; Discussion question: one or two fields for protocols or more? -SS 02May2022 |

We also highly encourage the readers of this GitHub to look for examples of submitted proteomic samples and analyses in **PRIDE: PRoteomics IDEntifications Database** [@pride], **PeptideAtlas** [@peptideatlas], **MassIVE: Mass Spectrometry Interactive Virtual Environment** [@massive], **iProX: Integrated Proteome Resources** [@iprox] and read the publication and view the corresponding repository of **jPOSTrepo: an international standard data repository for proteomes** [@jpostrepo].

# References